src/DocRaptor.php

```diff
@@ -39,15 +40,17 @@ class ApiWrapper
     protected $baseurl;
 
 
-
     /**
+     * @param HttpTransferInterface $httpClient
      * @param string|null $api_key
      */
-    public function __construct($api_key = null)
+    public function __construct(HttpTransferInterface $httpClient, $api_key = null)
```

Comment:
Unrelated to this branch, but what if api key was required and we threw a `MissingAPIKeyException` if it wasn't passed in?